packages/react-ui/src/types.d.ts

```diff
@@ -73,10 +73,6 @@ export type LegendWidgetUIData = {
   colors?: string[];
   labels?: (string | number)[];
   icons?: string[];
-  stats?: {
-    min: number | string;
-    max: number | string;
-  };
```

Comment:
breaking change here to document?

Reply:
`stats` wasn't even documented 😆

Reply:
![image](https://user-images.githubusercontent.com/9151432/154035605-0e2e2584-2377-4901-b991-e7fd14aaa55d.png)

Reply:
I'm ok with it, if not part of public api, 👍🏻